package.json

```diff
@@ -27,7 +27,7 @@
     "watch": "npm-run-all -p watch:*",
     "watch:css": "npm run build:css -- -w",
     "watch:js": "npm run build:js -- -w",
-    "prepublish": "not-in-install && npm run build && pkg-ok || in-install",
```

Comment:
I definitely don't have a deep understanding of the [in-publish](https://github.com/iarna/in-publish) package, but it looks like we are removing its usage here.

Can we remove in-publish as well?

Reply:
Yes! Good call.

Reply:
Even better call removing `not-prerelease` as well!